frontend/src/component/Search/CheckboxFilter.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { translate } from 'react-i18next'
+import PropTypes from 'prop-types'
+import { Icon } from 'tracim_frontend_lib'
+
+export const CheckboxFilter = props => {
+  const { props, state } = this
+  return (
+    <>
+      <div className='searchFilterMenu__content__item__title'>
```

Comment:
ideally, the classname of a component should look like the component name. This seems to be a children in a specific context
You could not have a Fragment at the root and a div with the classname CheckboxFilter

Reply:
fix